index.html

```diff
@@ -231,6 +231,43 @@
         instance, there are sufficient <a>installability signals</a> to warrant
         <a>installation</a> of the web application.
       </p>
+      <section data-dfn-for="AppBannerPromptOutcome">
+        <h4>
+          <code>AppBannerPromptOutcome</code> enum
+        </h4>
+        <pre class="idl">
+          enum AppBannerPromptOutcome {
+            "accepted",
+            "dismissed"
+          };
+        </pre>
+        <p>
+          The <dfn>AppBannerPromptOutcome</dfn> enum's values represent the
+          outcomes from <a data-lt="presents an install prompt">presenting the
+          end-user with an install prompt</a>.
+        </p>
+        <dl data-dfn-for="AppBannerPromptOutcome">
+          <dt>
+            <dfn>accepted</dfn>:
+          </dt>
+          <dd>
+            The end-user indicated that the user agent MUST attempt to
```

Comment:
This wording is a little strange. "The end-user indicated that <X>." This doesn't exactly convey that X should happen, since it's qualified by "the end-user indicated that". I don't think you should be mandating *what* the UA should do here; these are enum values after all, not algorithms. You should just describe what the enum represents (i.e., "The end-user indicated that they would like to install the web application.") Save the instruction for the UA for the "steps to install a web application" algorithm.